newrelic/hooks/hybridagent_opentelemetry.py

```diff
@@ -13,25 +13,74 @@
 # limitations under the License.
 
 import logging
+import os
 
 from newrelic.api.application import application_instance
 from newrelic.api.time_trace import add_custom_span_attribute, current_trace
-from newrelic.api.transaction import Sentinel, current_transaction
+from newrelic.api.transaction import current_transaction
 from newrelic.common.object_wrapper import wrap_function_wrapper
 from newrelic.common.signature import bind_args
 from newrelic.core.config import global_settings
 
 _logger = logging.getLogger(__name__)
 _TRACER_PROVIDER = None
 
+# Enable OpenTelemetry Bridge to capture HTTP 
+# request/response headers as span attributes:
+os.environ["OTEL_INSTRUMENTATION_HTTP_CAPTURE_HEADERS_SERVER_REQUEST"] = ".*"
```

Comment:
Is it safe to do this here? What if the customer is running otel but doesn't have our hybrid agent enabled?

Reply:
This logic is a placeholder until the configuration set up has been integrated